src/lib/ci.js

```diff
@@ -0,0 +1,47 @@
+
+export const CI_HOST = "https://lighthouse-dot-webdotdevsite.appspot.com";
+
+export async function fetchMedians() {
```

Comment:
Should we comment this out for now with a TODO that it's broken?

Reply:
I've just added a TODO. There's nothing inherently wrong with the call, I think?